src/elements.rs

```diff
@@ -569,7 +798,99 @@ named!(pub track_entry<&[u8], TrackEntry, Error>,
       })
     )
   )
-);
+);*/
+
+pub fn track_entry(input: &[u8]) -> IResult<&[u8], TrackEntry, Error> {
+    ebml_master(0xAE, |i| {
+        map(
+            pair(
+                permutation((
```

Comment:
This needs some comments. Why a pair with two permutations? Why not a single permutation?